src/NuGetSdkResolver.UnitTests/NuGet.MSBuildSdkResolver.UnitTests.csproj

```diff
@@ -0,0 +1,80 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{7FD51C62-1D6A-4DF5-9E70-E4D23CA3048C}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <RootNamespace>NuGet.MSBuildSdkResolver.UnitTests</RootNamespace>
+    <AssemblyName>NuGet.MSBuildSdkResolver.UnitTests</AssemblyName>
+    <IsTestProject>true</IsTestProject>
+  </PropertyGroup> -->
+
+  <PropertyGroup>
+    <TargetFrameworks>$(RuntimeOutputTargetFrameworks)</TargetFrameworks>
+    <PlatformTarget>$(RuntimeOutputPlatformTarget)</PlatformTarget>
+    <RootNamespace>NuGet.MSBuildSdkResolver.UnitTests</RootNamespace>
+    <AssemblyName>NuGet.MSBuildSdkResolver.UnitTests</AssemblyName>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <IsPackable>false</IsPackable>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
```

Comment:
I'd delete this line (allowing `EnableDefaultCompileItems` to be true), and then remove the `Compile` items below that are already in the project folder.  The rest of the projects don't consistently do this, but if we're adding a new project I think we may as well let it use the default globs.